etc/webapi.xml

```diff
@@ -48,6 +48,15 @@
             <resource ref="anonymous"/>
         </resources>
     </route>
+    <route method="POST" url="/V1/orders/:publicOrderId/quote/mine/authorizeAndPlace">
+        <service class="Bold\Checkout\Api\PlaceOrderInterface" method="authorizeAndPlace"/>
+        <resources>
+            <resource ref="self"/>
+        </resources>
+        <data>
+            <parameter name="quoteMaskId" force="true">%quote_mask_id%</parameter>
```

Comment:
TIL forcing request params in Adobe Commerce -> linking docs incase anyone else is unfamiliar https://developer.adobe.com/commerce/php/development/components/web-api/services/#forcing-request-parameters

Reply:
Thank you.